packages/apps-electron/src/electron.ts

```diff
@@ -44,15 +44,15 @@ const onReady = async () => {
 app.whenReady().then(onReady).catch(console.error);
 
 if (features.autoUpdater) {
-  autoUpdater.on('update-not-available', async () => {
-    await dialog.showMessageBox({
+  autoUpdater.on('update-not-available', () => {
+    // eslint-disable-next-line no-void
+    void dialog.showMessageBox({
```

Comment:
Doing `.catch(console.error)` on line 52 is probably more like what we use elsewhere. So swallow, but at least let the dev know something happenned. Or even `.catch(() => undefined)` if we really don't care at all.

Reply:
EDIT: Nothing wrong with the override, I just feel like I added way too much especially after the bump to typescript-eslint 3 (which is a bigger pain in the behind), so would like to avoid some even if we just swallow.

Reply:
Done.